solutions/standard/main.tf

```diff
@@ -232,3 +232,81 @@ data "ibm_database_connection" "existing_connection" {
   user_id       = data.ibm_database.existing_db_instance[0].adminuser
   user_type     = "database"
 }
+
+########################################################################################################################
+# Code Engine Kibana Dashboard instance
+########################################################################################################################
+
+data "http" "es_metadata" {
+  url      = "https://${local.es_username}:${local.es_password}@${local.es_host}:${local.es_port}"
+  insecure = true
+}
+
+locals {
+  es_host         = local.use_existing_db_instance ? data.ibm_database_connection.existing_connection[0].https[0].hosts[0].hostname : module.elasticsearch[0].hostname
+  es_port         = local.use_existing_db_instance ? data.ibm_database_connection.existing_connection[0].https[0].hosts[0].port : module.elasticsearch[0].port
+  es_username     = local.use_existing_db_instance ? data.ibm_database.existing_db_instance[0].adminuser : "admin"
+  es_password     = local.admin_pass
+  es_data         = jsondecode(data.http.es_metadata.response_body)
+  es_full_version = local.es_data.version.number
+}
+
+locals {
+  code_engine_project_name = "${var.prefix}-code-engine-kibana-project"
+  code_engine_app_name     = "${var.prefix}-kibana-app"
+}
+
+module "code_engine_kibana" {
```

Comment:
There should also be the option here to use a code engine project created from the code engine DA - so taking code engine details as input parameter.